maintainers.md

```diff
@@ -0,0 +1,24 @@
+# Maintainers
+
+## Current Maintainers As of 15th of September 2025
+
+- [Mitch Gaffigan](https://github.com/mgaffigan)
+- [Nico Piel](https://github.com/nicopiel)
```

Comment:
I want to start by saying that I'm highly appreciative of the amount of effort Nico has already contributed to working on the Gradle conversion.

However, I would like to see a bit more from him before granting maintainer rights. A primary responsibility of being a maintainer is helping to improve other people's PRs by giving thoughtful reviews with constructive feedback. If he's interested in the position, I'd love to see him start to review some of the open PRs.

I'm not approving Nico right now, but my vote is a "not yet" rather than a "no."